contents/blog/planning-a-company-offsite.md

```diff
@@ -0,0 +1,83 @@
+---
+date: 2022-08-08
+title: A startup's guide to planning an offsite in just 8 weeks
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/product-people.png
+featuredImageType: full
+author: ["grace-mckenzie"]
+categories: ["Inside PostHog", "Guides"]
+---
+
+As a company born at the beginning of the pandemic, PostHog has been fully-remote from its inception. Our team is 30+ people distributed across 10 countries; we have no central office, but [we still value getting together](/blog/asynchronous-remote-companies) in person. 
+
+We give employees [social and co-working budgets](/handbook/people/benefits) to encourage meetups and in-person onboarding. But the main way we facilitate in-person collaboration is [through offsites](/handbook/company/offsites) – small team offsites for focused work, and one all-company offsite per year for high-level strategic planning and cross-team interaction.  
+
+In large companies, offsite planning is centralized within the People or Operations teams, but this isn’t possible in many early-stage startups. This guide is all about how to plan a killer offsite in just eight weeks – including a few tips and tricks I’ve learned over a decade of organizing team retreats.
+
+> Grace McKenzie is Operations Manager at PostHog. Add more copy here
```

Comment:
@postgrace Would you like to write a short bio here? A couple of sentences would be ideal and we can link to your PH profile / LinkedIn etc.

Reply:
How's something like this?

"Grace McKenzie is the Operations Manager at PostHog, where she acts as a swiss-army-knife of sorts, tackling any and all back office projects that help build the scaffolding we need as our company scales. In her spare time, you can find her on the rugby pitch playing flanker or at her local billiards hall practicing her 8-ball breaks.